runtime/command_queue/enqueue_common.h

```diff
@@ -640,6 +640,8 @@ CompletionStamp CommandQueueHw<GfxFamily>::enqueueNonBlocked(
         gtpinNotifyPreFlushTask(this);
     }
 
+    DebugManager.log(DebugManager.flags.PrintDebugMessages.get(), "preemption = ", static_cast<int>(dispatchFlags.preemptionMode));
+    printDebugString(DebugManager.flags.PrintDebugMessages.get(), stdout, "preemption = %d.\n", static_cast<int>(dispatchFlags.preemptionMode));
```

Comment:
Please stick to one logging method, either via DebugManager or printDebugString.
I preffer to do this via DebugManager